doc/changelog.md

```diff
@@ -16,10 +16,16 @@ To be released at some future point in time
 Description
 
 - Update packaging dependency
```

Comment:
I would take out the changelog update, just bc we are not editing the changelog for PRs going into smartsim-refactor!

Reply:
Good point! I'll just opt the the `smartsim-refactor` branch out of the CI check